src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/Design/DesignerVerb.cs

```diff
@@ -6,6 +6,12 @@
 
 namespace System.ComponentModel.Design
 {
+    internal static partial class DesignerVerbRegex
+    {
+        [RegexGenerator(@"\(\&.\)")]
+        internal static partial Regex GetParameterReplacementRegex();
+    }
```

Comment:
We shouldn't need to introduce a new type for this.  GetParameterReplacementRegex can just be a private partial static on DesignerVerb, which can also be marked partial; the partial has no affect on the compiled metadata.